### DISCUSSION TITLE
Problem on starting Docker

### DISCUSSION THREAD

**Discussion by Lingua00** [larics/icuas24_competition#28] [General]
Hello, we have a problem with the ./docker_run.sh script. We installed Docker following your guide and then we started docker_build and docker_run. But as you can see from the screenshot it gives us problem, can you help us fixing it? Thank you!
![Screenshot from 2024-01-12 08-33-48](https://github.com/larics/icuas24_competition/assets/136790465/427ab805-4f36-4653-b5cf-c8f0b1eca5fe)

**Comment by fpetric** [comment]
This may be related to Docker installation. If you are using Docker Desktop, check here: https://github.com/NVIDIA/nvidia-docker/issues/1746

**Comment by Lingua00** [comment]
Hi! Thank you very much for your answer, we were able to start the
simulation by running the script with --focal-nogpu additional arguments.
I have one other question, I registered my group just today but I saw that
you sent some information to the group leader about the intermediate
submission. Is it possible to get that information? Thank you very much,
have a good day!

Linguerri Luca


Il giorno ven 12 gen 2024 alle ore 13:42 Frano Petric <
***@***.***> ha scritto:

**Reply by fpetric** [reply]
Yes, I will check the form over the weekend and forward the info to the newly registered teams.
